docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,285 @@
+# HVAC_SystemZoneAssignment - Rule 11-10
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** "The service water heating systems type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2"  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each SWH bat  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_connected_to_use_type()
+2. get_SHW_types_and_SHW_use()
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are SHW loads
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW use types and SHW use in the building: `shw_bats_and_equip_dict = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)`
+
+- call the function get_SHW_types_and_SHW_use for the proposed: `shw_bat_and_use_dict_p = get_SHW_types_and_SHW_use(P_RMD)`
+- look through each of the SHW bats in the building: `for shw_bat in shw_bat_and_use_dict_p:`
+  - check if there are SHW loads in this use type, continue to rule logic: `if len(shw_bat_and_use_dict_p[shw_bat]) > 0: CONTINUE TO RULE LOGIC`
+  - otherwise, rule is not applicable for this shw_bat: `else: NOT_APPLICABLE`
+
+
+    ## Rule Logic:
+  - create a value to track the rule status: `rule_status = "FAIL"`
+  - create a variable to track any notes: `rule_note = """`
+  - there should be one SHW Equipment for this space type: `if len(shw_types_and_use_dict[shw_bat]["SHWHeatingEq"]) != 1:`
```

Comment:
multiple `ALL_OTHERS`?

Reply:
@JacksonJ-KC I had the same thought. I think it is a grey area as to whether there could be multiple All_Other. Godo topic for the call.